tdrs-frontend/nginx/local/locations.conf

```diff
@@ -0,0 +1,51 @@
+#location ~ \.(css)$ {
+#    add_header  Content-Type    text/css;
+#}
+
+#location = /nginx_status {
+#    stub_status on;
+#    access_log   off;
+#    deny all;
+#}
+
+location ^~ /v1/ {
+        proxy_pass              http://${BACK_END}:8080/v1/;
+        proxy_set_header   Host $host;
+        proxy_set_header   X-Real-IP         $remote_addr;
+        proxy_set_header   X-Forwarded-For   $proxy_add_x_forwarded_for;
+        proxy_set_header   X-Forwarded-Proto https;
+
+        proxy_connect_timeout 60;
+        proxy_send_timeout    60;
+        proxy_buffer_size     4k;
+        proxy_temp_file_write_size 64k;
+}
+
+location = /profile {
+    index index.html index.htm;
+    try_files $uri $uri/ /index.html;
+}
+
+location = /home {
+    index index.html index.htm;
+    try_files $uri $uri/ /index.html;
+}
+
+location ^~ /data-files {
+    index index.html index.htm;
+    try_files $uri $uri/ /index.html;
+}
+
+location = /login {
+    index index.html index.htm;
+    try_files $uri $uri/ /index.html;
+}
+
+location = / {
+    index index.html index.htm;
+    try_files $uri $uri/ /index.html;
+}
```

Comment:
Can these `locations` be collapsed into one block? They appear identical.

Reply:
it can be done for sure. An easy fix

The before after missing is because of file relocation -> moved some of the files to another directory

Reply:
> The before after missing is because of file relocation -> moved some of the files to another directory

Did you just manually move it vs using `git mv`? Minor nit

Reply:
> > Virgin Islands
> 
> I bet the test will fail because I will have to add the following lists:
> 
> https://lite.ip2location.com/virgin-islands-(u.s.)-ip-address-ranges https://lite.ip2location.com/guam-ip-address-ranges?lang=en_US



> > Virgin Islands
> 
> I bet the test will fail because I will have to add the following lists:
> 
> https://lite.ip2location.com/virgin-islands-(u.s.)-ip-address-ranges https://lite.ip2location.com/guam-ip-address-ranges?lang=en_US

Added both location to whitelist